api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -21,23 +21,27 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-package net.kyori.text.util;
+package net.kyori.adventure.audience;
 
-import java.util.Map;
+import net.kyori.adventure.sound.Sound;
+import net.kyori.adventure.text.Component;
 import org.checkerframework.checker.nullness.qual.NonNull;
 
-@Deprecated
-public interface ToStringer {
+/**
+ * An audience.
+ */
+public interface Audience {
+  /**
+   * Sends a message.
+   *
+   * @param message the message
+   */
+  void message(final @NonNull Component message);
```

Comment:
`sendMessage` and `playSound`? 

I know the full lowercase thing is a style, but it just sounds weird rn. It would also line up nicely with platform APIs.

Reply:
`sound` changed to `playSound`, but I think `message` alone is fine.

Reply:
I would still strongly consider having a `sendMessage` to allow for easier drop-in replacement. Maybe `message` is an alias?